src/packages/cli/src/utils/experimental.ts

```diff
@@ -0,0 +1,11 @@
+import chalk from 'chalk'
```

Comment:
I copied this over from the `migrate` package. Maybe I should instead export it from Migrate and import it here?

Studio will likely go GA early September, so we can probably delete this file then?

Reply:
Looks good like this to me, we will remove it when it's not experimental anymore